frontend/src/Editor/ManageAppUsers.jsx

```diff
@@ -18,6 +18,7 @@ class ManageAppUsers extends React.Component {
       app: { ...props.app },
       slugError: null,
       isLoading: true,
+      isRequestInProgress: false,
```

Comment:
This must be confusing as this component have multiple requests. Can we rename this to `isSlugVerificationInProgress` or something better?